src/internal/plugins/helpers/runtime-content-wrapper.tsx

```diff
@@ -1,41 +0,0 @@
-// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
-// SPDX-License-Identifier: Apache-2.0
-import React, { useContext, useEffect, useRef } from 'react';
-
-import { ActiveDrawersContext } from '../../../app-layout/utils/visibility-context';
-import { MountContentContext } from '../controllers/drawers';
-
-type VisibilityCallback = (isVisible: boolean) => void;
-
-interface RuntimeContentWrapperProps {
-  mountContent: (container: HTMLElement, mountContent: MountContentContext) => void;
-  unmountContent: (container: HTMLElement) => void;
-  id?: string;
-}
-
-export function RuntimeContentWrapper({ mountContent, unmountContent, id }: RuntimeContentWrapperProps) {
-  const ref = useRef<HTMLDivElement>(null);
-  const visibilityChangeCallback = useRef<VisibilityCallback | null>(null);
-  const activeDrawersIds = useContext(ActiveDrawersContext);
-  const isVisible = !!id && activeDrawersIds.includes(id);
```

Comment:
This is the problematic code. It was unintentionally included into the flashbar runtime API too, which is unexpected